Serilog.Sinks.Scalyr/KeyValuePairExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Serilog.Sinks.Scalyr;
+
+/// <summary>
+///     Extension methods for <see cref="KeyValuePair{TKey,TValue}"/>.
+/// </summary>
+public static class KeyValuePairExtensions
+{
+    /// <summary>
+    /// Deconstructs the key value pair into a key and value.
+    /// </summary>
+    /// <param name="kvp">The key value pair.</param>
+    /// <param name="key">The key.</param>
+    /// <param name="value">The value.</param>
+    /// <typeparam name="TK">The type of the key.</typeparam>
+    /// <typeparam name="TV">The type of the value.</typeparam>
+    public static void Deconstruct<TK, TV>(this KeyValuePair<TK, TV> kvp, out TK key, out TV value)
```

Comment:
Where is this being used?

Reply:
https://github.com/TinyBlueRobots/Serilog.Sinks.Scalyr/pull/4/files#diff-e0f14e9c1cf87a7cb5a717977c3cd84d37e63d51dc67adffbb5f387800bd5365R46

If you'd like, I can remove it. It's not strictly necessary or anything. :)

Reply:
Oh I see, it's a custom deconstructor, I didn't know C# could do this 👍